examples/step_operator_remote_training/config.yaml

```diff
@@ -0,0 +1,15 @@
+name: step_operator_pipeline
```

Comment:
for other examples you mentioned in the README the option to run with `-c config`, why not for this one?